### PR TITLE
Added more things to the Surveyor

### PR DESCRIPTION
**Pull request by Eahlstan**
High metal content worlds and metal-rich bodies can now be shown.
Bodies with a high eccentricity of > 0.95 can now be shown.

### CODE REVIEW

File: EDDiscovery/UserControls/Overlays/UserControlSurveyor.cs [hunk 1]
```diff
@@ -409,9 +418,14 @@ private string InfoLine(ISystem sys, StarScan.ScanNode sn, JournalScan js)
...
+            information.Append((js.PlanetTypeID == EDPlanet.High_metal_content_body && !js.Terraformable) ? @" is a high metal content world." : null);
+            information.Append((js.PlanetTypeID == EDPlanet.Metal_rich_body && js.Terraformable) ? @" is a terraformable metal-rich body." : null);
+            information.Append((js.PlanetTypeID == EDPlanet.Metal_rich_body && !js.Terraformable) ? @" is a metal-rich body." : null);
+            information.Append((js.Terraformable && !js.WaterWorld && js.PlanetTypeID != EDPlanet.High_metal_content_body && js.PlanetTypeID != EDPlanet.Metal_rich_body) ? @" is a terraformable planet.".T(EDTx.UserControlSurveyor_isaterraformableplanet) : null);
             information.Append((js.HasRings) ? @" Has ring.".T(EDTx.UserControlSurveyor_Hasring) : null);
             information.Append((js.HasMeaningfulVolcanism) ? @" Has ".T(EDTx.UserControlSurveyor_Has) + js.Volcanism + "." : null);
+            information.Append((js.nEccentricity >= 0.95) ? @"Has a high eccentricity of " + (js.nEccentricity??"Unknown") : null);
```
Comment (klightspeed) [inline EDDiscovery/UserControls/Overlays/UserControlSurveyor.cs:428]: error CS0019: Operator '??' cannot be applied to operands of type 'double?' and 'string'

Comment (Eahlstan) [reply EDDiscovery/UserControls/Overlays/UserControlSurveyor.cs:428]: Strange, Visual Studio didn't complain at my side oO

Comment (Eahlstan) [reply EDDiscovery/UserControls/Overlays/UserControlSurveyor.cs]: Guess Robbys double paranoia isn't needed anyways as it won't work if nEccentricity is empty.

File: EDDiscovery/UserControls/Overlays/UserControlSurveyor.cs [hunk 2]
```diff
@@ -409,9 +418,14 @@ private string InfoLine(ISystem sys, StarScan.ScanNode sn, JournalScan js)
             information.Append((js.Earthlike) ? @" is an earth like world.".T(EDTx.UserControlSurveyor_isanearthlikeworld) : null);
             information.Append((js.WaterWorld && !js.Terraformable) ? @" is a water world.".T(EDTx.UserControlSurveyor_isawaterworld) : null);
             information.Append((js.WaterWorld && js.Terraformable) ? @" is a terraformable water world.".T(EDTx.UserControlSurveyor_isaterraformablewaterworld) : null);
-            information.Append((js.Terraformable && !js.WaterWorld) ? @" is a terraformable planet.".T(EDTx.UserControlSurveyor_isaterraformableplanet) : null);
+            information.Append((js.PlanetTypeID == EDPlanet.High_metal_content_body && js.Terraformable) ? @" is a terraformable high metal content world." : null);
```
Comment (robbyxp1) [inline EDDiscovery/UserControls/Overlays/UserControlSurveyor.cs:421]: these are missing those all very important .t(EDTX.. )

make up a new applicable ID in the .cs, and add a entry to the example-ex file (translation-example-uc.tlp) in the right section!

Comment (Eahlstan) [reply EDDiscovery/UserControls/Overlays/UserControlSurveyor.cs]: Yes Sir!

**Pull request closed without merging**